qt_ui/windows/basemenu/QRecruitBehaviour.py

```diff
@@ -58,7 +58,8 @@ def add_purchase_row(self, unit_type: Type[UnitType], layout: QLayout,
         existing_units = self.cp.base.total_units_of_type(unit_type)
         scheduled_units = self.pending_deliveries.units.get(unit_type, 0)
 
-        unitName = QLabel("<b>" + db.unit_type_name_2(unit_type) + "</b>")
+        #unitName = QLabel("<b>" + db.unit_type_name_2(unit_type) + "</b>")
```

Comment:
Just delete commented out code. We have version control :)

Reply:
Didn't even spot that one. Thanks, fixed.